src/tests/HDFFile.cxx

```diff
@@ -7,10 +7,33 @@
 #include "../helper.h"
```

Comment:
Yes, I mean you have two files called `HDFFile.cxx`. I think that can lead to confusion, particularly as it is usual to navigate around a codebase by typing file or class names.